.circleci/config.yml

```diff
@@ -583,9 +586,11 @@ jobs:
       - store_artifacts: *artifacts_test_results
 
   b_ems:
+    resource_class: xlarge
     docker:
       - image: ethereum/solidity-buildpack-deps:emsdk-1.39.15-2
     environment:
+      NJOBS: 9
```

Comment:
In https://github.com/ethereum/solidity/pull/9368#discussion_r454183010 @ekpyron suggested using `MAKEFLAGS` for that.

Reply:
At least if it works, yeah :-).